db/migrations/004_incidents.sql

```diff
@@ -195,6 +197,22 @@ CREATE TRIGGER incident_to_incident_history
     FOR EACH ROW
     EXECUTE PROCEDURE insert_incident_created_in_incident_history();
 
+-- Insert incident status updates in incident_histories
+CREATE OR REPLACE FUNCTION insert_incident_status_update_in_incident_history()
+RETURNS TRIGGER AS $$
+BEGIN
+    INSERT INTO incident_histories(incident_id, created_by, type, description) VALUES (NEW.id, NEW.created_by, 'incident.status_updated', NEW.status);
```

Comment:
I don't think NEW.created_by is the correct value - 

Long term we want to link the oauth tokens generated by kratos proxy into https://postgrest.org/en/v10.0/auth.html#web-users-sharing-role so that we can get the identity of the user logged in, so maybe we should create a get_current_user() function and In then in the job set a local session variable = System, and have the function look at the postgrest session variable and then fallback to System user